src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Embedded.cs

```diff
@@ -10,70 +10,6 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests
 {
     public class AttributeTests_Embedded : CSharpTestBase
     {
-        [Fact]
-        public void EmbeddedAttributeNamespace()
-        {
-            var code = @"
-namespace Microsoft.CodeAnalysis.EmbeddedAttribute
-{
-    class C { }
-}
-namespace TestReference
-{
-    [Microsoft.CodeAnalysis.Embedded]
-    internal class TestType1 { }
-
-    [Microsoft.CodeAnalysis.EmbeddedAttribute]
-    internal class TestType2 { }
-
-    internal class TestType3 { }
-}";
-
-            CreateCompilation(code).VerifyEmitDiagnostics(
-                // (11,29): error CS0616: 'Microsoft.CodeAnalysis.EmbeddedAttribute' is not an attribute class
-                //     [Microsoft.CodeAnalysis.EmbeddedAttribute]
-                Diagnostic(ErrorCode.ERR_NotAnAttributeClass, "EmbeddedAttribute").WithArguments("Microsoft.CodeAnalysis.EmbeddedAttribute").WithLocation(11, 29),
-                // (8,29): error CS0616: 'Microsoft.CodeAnalysis.EmbeddedAttribute' is not an attribute class
-                //     [Microsoft.CodeAnalysis.Embedded]
-                Diagnostic(ErrorCode.ERR_NotAnAttributeClass, "Embedded").WithArguments("Microsoft.CodeAnalysis.EmbeddedAttribute").WithLocation(8, 29)
-                );
-        }
-
-        [Fact]
-        public void EmbeddedAttributeInMergedNamespaceSymbol()
-        {
-            var reference_cs = @"
-namespace Microsoft.CodeAnalysis
-{
-    internal class EmbeddedAttribute : System.Attribute { }
-}";
-            var reference = CreateCompilation(new[] { reference_cs });
-            reference.VerifyDiagnostics();
-
-            var comp_cs = @"
-namespace Microsoft.CodeAnalysis.EmbeddedAttribute
-{
-    class C { }
-}";
-            var comp = CreateCompilation(new[] { comp_cs }, options: WithNonNullTypesTrue(), references: new[] { reference.ToMetadataReference() });
-            comp.VerifyEmitDiagnostics(
-                // error CS0518: Predefined type 'Microsoft.CodeAnalysis.EmbeddedAttribute' is not defined or imported
-                Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound).WithArguments("Microsoft.CodeAnalysis.EmbeddedAttribute").WithLocation(1, 1)
-                );
-
-            var type = comp.GetWellKnownType(WellKnownType.Microsoft_CodeAnalysis_EmbeddedAttribute);
-            Assert.True(type.IsErrorType());
-
-            // https://github.com/dotnet/roslyn/issues/29683 CSharpCompilation.AbstractSymbolSearcher needs to inject namespaces and types too
```

Comment:
>https://github.com/dotnet/roslyn/issues/29683 [](start = 15, length = 45)

can be closed?

Reply:
> can be closed?

I think it can be, I'll add to the list.

---
In reply to: [234738966](https://github.com/dotnet/roslyn/pull/31212#discussion_r234738966) [](ancestors = 234738966)